pallets/beefy-light-client/src/benchmarking.rs

```diff
@@ -30,7 +30,232 @@
 
 use super::*;
 
-// #[allow(unused)]
-// use crate::Pallet as BeefyLightClient;
-// use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, whitelisted_caller};
-// use frame_system::RawOrigin;
+use crate::benchmark_features::*;
+use crate::test_helpers::*;
+use crate::Pallet as BeefyLightClient;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+use hex_literal::hex;
+
+benchmarks! {
+    initialize {
+        let root = hex!("36ee7c9903f810b22f7e6fca82c1c0cd6a151eca01f087683d92333094d94dc1");
+        let curr_val_set = ValidatorSet {
+            id: 0,
+            len: 3,
+            root: root.into(),
+        };
+        let next_val_set = ValidatorSet {
+            id: 1,
+            len: 3,
+            root: root.into(),
+        };
+    }: _(RawOrigin::Root, SubNetworkId::Mainnet, 1, curr_val_set, next_val_set)
+    verify {
+        assert!(BeefyLightClient::<T>::current_validator_set(SubNetworkId::Mainnet).is_some());
+        assert!(BeefyLightClient::<T>::next_validator_set(SubNetworkId::Mainnet).is_some());
+    }
+
+
+    submit_signature_commitment_10_128 {
```

Comment:
What is output of your benchmarks?
As I see the result is not mapped to WeightInfo trait.
Also how did you get submit_signature_commitment() weight? Because you will get of methods submit_signature_commitment_x_y() with different weights?